frontend/lib/src/components/widgets/DataFrame/columns/SelectboxColumn.ts

```diff
@@ -85,7 +86,6 @@ function SelectboxColumn(props: BaseColumnProps): BaseColumn {
           .map(opt => toSafeString(opt)), // convert everything to string
       ],
       value: "",
-      readonly: !props.isEditable,
```

Comment:
This argument was unnecessary. It needs to be at the top level and it is already added there.